README.md

```diff
@@ -30,7 +30,7 @@ php artisan filament-kanban:install
 
 ## Before You Start
 
-> [!IMPORTANT]  
```

Comment:
This line for example doesn't have to change for this PR.

Reply:
> This line for example doesn't have to change for this PR.

Oh, I understand now. However, it's my code editor that automatically trims additional spaces. If that's a problem I'll try disabling it for the MD file edition.